src/main/java/org/xstefank/webhook/WebHookEndpoint.java

```diff
@@ -28,23 +29,20 @@
 
     private static final Logger log = Logger.getLogger(WebHookEndpoint.class);
     private static final FormatConfig config = readConfig();
+    private static final WhitelistProcessing whitelistProcessing =
+            WhitelistProcessing.IS_WHITELISTING_ENABLED ? new WhitelistProcessing(config) : null;
+
     private TemplateChecker templateChecker = new TemplateChecker(config);
 
     @POST
     @Path("/pull-request")
     @Consumes(MediaType.APPLICATION_JSON)
-    public void processPullRequest(JsonNode payload) {
-        if (!SkipCheck.shouldSkip(payload, config)) {
-            String errorMessage = templateChecker.checkPR(payload);
-            if (errorMessage != null) {
-                log.info("updating status");
-
-                GitHubAPI.updateCommitStatus(config.getRepository(),
-                        payload.get(Utils.PULL_REQUEST).get(Utils.HEAD).get(Utils.SHA).asText(),
-                        errorMessage.isEmpty() ? CommitStatus.SUCCESS : CommitStatus.ERROR,
-                        config.getStatusUrl(),
-                        errorMessage.isEmpty() ? "valid" : errorMessage, "PR format check");
-            }
+    public void processRequest(JsonNode payload) {
+        if (whitelistProcessing != null &&
+                payload.has(Utils.ISSUE)) {
+            whitelistProcessing.process(payload);
```

Comment:
```suggestion
            whitelistProcessing.processComment(payload);
```
maybe this could better indicate that it's going to try to match comment commands?